src/sentry/replays/usecases/ingest/dom_index.py

```diff
@@ -131,7 +134,10 @@ def get_user_actions(
 
         if event.get("type") == 5 and event.get("data", {}).get("tag") == "breadcrumb":
             payload = event["data"].get("payload", {})
-            if payload.get("category") == "ui.click":
+            category = payload.get("category")
+            if category == "ui.slowClickDetected":
+                logger.info(f"Slow click detected: {json.dumps(payload)}")
```

Comment:
it might be worth logging it as structured json like so:

```
logger.info("slow click detected",extra=payload)
```
probably also worth adding the project_id and replay_id to this log so its easy to go watch the replay after too to see what it looked like